packages/pcd/gpc-pcd/src/util.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  PODName,
+  PODRawValue,
+  POD_NAME_REGEX,
+  podValueFromRawValue,
+  podValueToRawValue
+} from "@pcd/pod";
+import JSONBig from "json-bigint";
+import { PODEntryRecord } from "./GPCPCD";
+
+const jsonBigSerializer = JSONBig({
+  useNativeBigInt: true,
+  alwaysParseAsBig: true
+});
+
+/**
+ * Deserializes `PODEntryRecord` from the simplified format produced by
+ * {@link podEntryRecordToSimplifiedJSON}.  Type information is inferred from
+ * the values in a way which should preserve hashing and circuit behavior, but
+ * isn't guaranteed to be identical to the types before serialization.  For
+ * instance, small numbers are always annotated as `int`, rather than
+ * `cryptographic`.
+ *
+ * @param simplifiedJSON a string representation of `PODEntryRecord`
+ * @returns `PODEntryRecord` deserialized from the string
+ * @throws if the serialized form is invalid
+ */
+export function podEntryRecordFromSimplifiedJSON(
```

Comment:
Not your fault in this PR, but I dislike how you're forced to reimplement parts of the existing PODEntries <-> SimplifiedJSON serialization here.  Serializing to/from strings doesn't compose nicely.  I'm taking this as another piece of evidence that we need to build a serialization method which produces a JSON-compatible object which can be serialized with JSON.serialize() instead.   That's high on my list of things to do when I find some time.